Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Helpers/Help.cs

```diff
@@ -137,6 +152,15 @@ public static bool ReportFileExist(string reportFolderPath)
 						return true;
 					}
 				}
+				else
+				{
+					var fileName = Path.GetFileName(Path.GetDirectoryName(reportFolderPath));
+					if (!string.IsNullOrEmpty(fileName) && fileName.Contains("ProjectFiles"))
+					{
+						fileName = fileName.Substring(0, fileName.LastIndexOf(".ProjectFiles", StringComparison.Ordinal));
+					}
+					_messageBoxService.ShowInformationMessage($@"Please run the Analyze File batch task for the project ""{fileName}"", otherwise it will not be included within the Export Analysis Reports", "Informative message");
```

Comment:
This string should be placed in resources for localization purposes

Reply:
I've moved all the messages within the PluginResources.resx file